src/main/java/ca/uhn/fhir/jpa/starter/ResourceProvider/PatientResourceProvider.java

```diff
@@ -23,12 +21,13 @@
 @Component
 public class PatientResourceProvider implements IResourceProvider {
 
-	private static final Logger log = LoggerFactory.getLogger(PatientResourceProvider.class);
```

Comment:
Is logger never used? Why was it never use? should it be use?

Reply:
Added more logging